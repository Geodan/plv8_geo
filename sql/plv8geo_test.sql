CREATE SCHEMA plv8;
CREATE EXTENSION postgis;
CREATE EXTENSION plv8;
DO $$
  plv8.elog(WARNING, 'plv8.version = ' + plv8.version); // Will output the PL/v8 installed as a PostgreSQL `WARNING`.
$$ LANGUAGE plv8;
CREATE EXTENSION plv8geo;
select plv8.plv8_startup();
--do language plv8 'load_module("delaunator")';
--SELECT plv8.delaunator([[1,1],[10,10],[5,5]]) AS values FROM foo;
