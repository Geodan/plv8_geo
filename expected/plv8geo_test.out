CREATE SCHEMA plv8;
CREATE EXTENSION postgis;
CREATE EXTENSION plv8;
CREATE EXTENSION plv8geo;
select plv8.plv8_startup();
 plv8_startup 
--------------
 
(1 row)

do language plv8 ' load_module("d3"); ';
do language plv8 ' load_module("d3-geo"); ';
do language plv8 ' load_module("d3-force"); ';
do language plv8 ' load_module("d3-hexbin"); ';
do language plv8 ' load_module("d3-contour"); ';
do language plv8 ' load_module("delaunator"); ';
do language plv8 ' load_module("topojson"); ';
/**
do language plv8 $$
	var topo = topojson.topology({entities: {
		type: 'Feature',
		geometry: {
			type: 'Point',
			coordinates: [0,0]
		}
	}},1);
	plv8.elog(NOTICE,JSON.stringify(topo));
$$;
**/
